disnake/enums.py

```diff
@@ -393,6 +393,9 @@ class AuditLogAction(Enum):
     automod_block_message                 = 143
     automod_send_alert_message            = 144
     automod_timeout                       = 145
+    onboarding_question_create            = 163
+    onboarding_question_update            = 164
```

Comment:
Since everything else is called "prompt", these should also be `onboarding_prompt_create`/`_update`

Reply:
3f4be9542c46304f1a2d515b6874af8c58364a2a